docs/index.md

```
@@ -0,0 +1,137 @@
+```
+
+### Authentication
+This API uses JWT (JSON Web Tokens) for authentication. Include the token in the Authorization header:
+
+```
+Authorization: Bearer <your-jwt-token>
+```
+
+## 📚 API Endpoints
+
+### Health Check
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)